specification/Metadata/README.md

```diff
@@ -21,283 +21,240 @@ Draft
+Names (`name`) provide a human-readable label for a property, and must be unique to a property within a class. Names must be valid UTF-8 strings, and should be written in natural language. Property names do not have inherent meaning; to provide such a meaning, a property must also define a [semantic](#semantics).
 
-An array property must declare a component type, which is the type of each component in the array. A component type may be any type from the table above with the exception of `ARRAY`; arrays of arrays are not supported.
+> **Example:** A typical ID / Name pair, in English, would be `localTemperature` and `"Local Temperature"`. In Japanese, the name might be represented as "きおん". Because IDs are restricted to alphanumeric characters and underscores, use of helpful property names is essential for clarity in many languages.
 
-A component count denotes the number of components for fixed-length arrays. It must be a value greater than one. If a component count is not specified the array is considered to be a variable-length array. Variable-length arrays may be any length, including zero-length.
+> **Example:**
 
-Arrays can be used to represent vector and matrix properties as shown in the example below.
+#### Description
 
-```jsonc
-{
-  "schema": {
-    "classes": {
-      "car": {
-        "properties": {
-          "forwardDirection": {
-            "description": "A forward direction vector",
-            "type": "ARRAY",
-            "componentType": "FLOAT64",
-            "componentCount": 3
-          },
-          "passengers": {
-            "name": "Passenger Names",
-            "description": "There are a variable number of passengers because componentCount is undefined.",
-            "type": "ARRAY",
-            "componentType": "STRING",
-          },
-          "modelMatrix": {
-            "description": "A 4x4 model matrix",
-            "type": "ARRAY",
-            "componentType": "FLOAT32",
-            "componentCount": 16
-          }
-        }
-      }
-    }
-  }
-}
-```
+Descriptions (`description`) provide a human-readable explanation of a property, its purpose, or its contents. Typically at least a phrase, and possibly several sentences or paragraphs. To provide a machine-readable semantic meaning, a property must also define a [semantic](#semantics).
 
-### Enums
+#### Semantic
 
-Enum properties are denoted by the `ENUM` type. An enum property must provide the ID of the specific enum it uses.
+Property IDs, names, and descriptions do not impute meaning. To provide such a meaning, properties may be assigned a semantic identifier string (`semantic`), indicating how the property's content should be interpreted. Semantic identifiers may be defined by the [Cesium Metadata Semantic Reference](./Semantics/) or by external semantic references, and may be application-specific. Identifiers should be uppercase, with underscores as word separators.
```

Comment:
Comparing the Semantics link to the rest of the document, it looks like we've been inconsistent with the links. I _think_ it's `./Semantics/` like you have here, but good to double check.